codespell_lib/data/dictionary_code.txt

```diff
@@ -39,5 +39,7 @@ storaget->storage
 subprocess'->subprocess's
 thead->thread
 todays->today's
+udate->update
+udates->updates
```

Comment:
Yeah as @sebweb3r said:
```suggestion
udate->update, date,
udates->updates, dates,
```

Reply:
@peternewman @sebweb3r Thanks for your review.

Please could you elaborate why you think that `udate` could be a misspelling of `date`?

The `udate` misspelling of `update` for example is originating from some one (i have done this typo myself already a few times) by not (fully) hitting the `p` button while typing.

But typing `udate` instead of `date` would require to hit one additional button `u` which is not even near the `d` key on the keyboard.

I don't have any strong reasons against not adding these suggestions but would first like to understand the rationale behind these suggestions.

Thanks

Reply:
Hey, 

f.e. "leftovers" from the word in front. Date is just a very common word, that is only one (extra) letter away.

I just had a look on [grep.app](https://grep.app/search?current=64&q=udate&words=true) to look if that mistake happened.
It looks like udate itself is a quite common variable name. Thus, udate should go to the code dictionary.

Reply:
Thanks for the explanation. As i don't have any strong opinion about that i have submitted the suggestions in 2a9f5c3ecfa70556e03e2a8c45ef9fe3f3374f02

Note that `udate` and `udates` is / was already added to the code dictionary from the beginning.

Reply:
Oh. Mea culpa. It seems like I was not yet awake this morning.

Reply:
E.g. "people you date" becomes "people yo udate". The other part of the reasoning is to make sure that automatic mode doesn't leave us with "people yo update" without warning you. In comparison, "comparrison" as I just typed is pretty much guaranteed to be a typo for comparison so wouldn't particularly benefit from other suggestions. I guess broadly the longer the word the greater the chance you can be confident it's only got one logical correction.

Reply:
@peternewman Thank you very much for taking the time to explain this in such a level of detail. This really explains a lot :+1: